scripts/test/nightly/runTests.sh

```diff
@@ -31,20 +31,22 @@ fi
 VPC=$1
 SUBNETS=$2
 RESULTS_BUCKET=$3
-if [ "$4" = "performance" ]; then
+if [ "$4" == "performance" ]; then
   EXTRA_MAVEN_PARAMS="-Dsleeper.system.test.cluster.enabled=true"
-else
+  TEST_NAME="performance"
+elif [ "$4" == "functional" ]; then
   EXTRA_MAVEN_PARAMS="-Dsleeper.system.test.cluster.enabled=false"
+  TEST_NAME="functional"
+else
+  echo "Invalid test type: $4"
+  echo "Valid test types are: performance, functional"
+  exit 1
```

Comment:
The usage message still says the test type is optional and defaults to functional. It looks like this will fail if only 3 arguments are given?

Reply:
Thanks, fixed